docs/buffer/overview.md

```diff
@@ -0,0 +1,47 @@
+# Buffer Architecture Overview
+
+- A Proxy API to buffer collected data and their import to Storage tables in batches.
```

Comment:
`A Proxy API` - neviem ci to je dost jednoznacne.

Mozno take nieco?
- The Buffer Service consists of API and Worker nodes.
- The API nodes:
  - Contain configuration endpoints.
  - Collect data via the import endpoint.
- The Worker nodes import the collected data into Storage tables in batches.

Reply:
👍